components/function-controller/internal/controllers/kubernetes/shared.go

```diff
@@ -12,11 +12,13 @@ const (
 	ConfigLabel              = "serverless.kyma-project.io/config"
 	RuntimeLabel             = "serverless.kyma-project.io/runtime"
 	RbacLabel                = "serverless.kyma-project.io/rbac"
+	ManagedByLabel           = "serverless.kyma-project.io/managed-by"
```

Comment:
What I meant was to use `FunctionManagedByLabel` from `pkg/apis/serverless/v1alpha1`, because they have the same value. I know the rest of those labels connected with kubernetes resoruces reside here, but it's probably a bad thing, as they're not exported at all :/